src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Set up iOS development
+description: >-
+  Configure your development environment to
+  run, build, and deploy Flutter apps for iOS devices.
+short-title: Set up iOS
+---
+
+Learn how to set up your development environment
+to run, build, and deploy Flutter apps for iOS devices.
+
+:::note
+If you haven't set up Flutter already,
+visit and follow the [Get started with Flutter][] guide first.
+
+If you have already installed Flutter,
+ensure that it's [up to date][].
+:::
+
+[Get started with Flutter]: /get-started/install
+[up to date]: /install/upgrade
+
+[up to date]: /install/upgrade
+
+## Install Xcode
+
+With Xcode, you can run Flutter apps on
+an iOS physical device or on the iOS Simulator.
+
+If you haven't done so already,
+install and set up [Xcode][].
+If you have installed Xcode, ensure that
+it's up to date and continue to the next step.
+(After setting up the command-line tools
+and accepting the license, as shown here.)
+
+[Xcode]: https://developer.apple.com/xcode/
+
+To configure Xcode's command-line tools,
+use the following command:
+
+```console
+$ sudo sh -c 'xcode-select -s /Applications/Xcode.app/Contents/Developer && xcodebuild -runFirstLaunch'
+```
```

Comment:
This part doesn't require sudo, but `xcodebuild -license` does. The three commands are:
```
$ xcode-select -s /Applications/Xcode.app
$ xcodebuild -runFirstLaunch
$ sudo xcodebuild -license
```
I also think `xcodebuild -downloadPlatform iOS` is required for both physical devices and simulator, so you can add it to the Xcode section instead of in the "Virtual device" section.